node/hack/governor/src/index.ts

```diff
@@ -44,15 +63,22 @@ axios
             if (addr !== "*") {
                 let data = res.data.AllTime[chain][addr]
                 let notional = parseInt(data.Notional)
-                if (notional > MinNotional) {
-                  if (data.Address == "ust") {
-                    continue
-                  }
+                if ((notional > MinNotional) || (includedTokens.has(chain + ":" + data.Address.toLowerCase()))) {
                   let chainId = parseInt(chain) as ChainId
-                  const wormholeAddr = tryNativeToHexString(
-                    data.Address,
-                    chainId
-                  );
+                  let wormholeAddr: string
+                  try {
+                    wormholeAddr = tryNativeToHexString(
+                      data.Address,
+                      chainId
+                    );
+                  } catch (e) {
+                    if (chainId === CHAIN_ID_ALGORAND && data.Address === "algo") {
+                      wormholeAddr = "0x0000000000000000000000000000000000000000"
+                    } else {
+                      console.log("Ignoring symbol \"" + data.Symbol + "\" because the address \"" + data.Address + "\" is invalid")
```

Comment:
A bit easier to read:

```suggestion
                      console.log(`Ignoring symbol '${data.Symbol}' because the address '${data.Address}' is invalid`)
```

Reply:
Changed.